script/install/xcode

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+set -e
+
+INSTALL_SCRIPTS_PATH="$(cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd -P)"
+source "$INSTALL_SCRIPTS_PATH/helper_functions"
+
+install_xcode() {
+  # Install xcode so we can set the command line tools location, which is
+  # necessary for `brew bundle`.
+  # TODO: mas installer should be an idempotent function
+  mas install 497799835
```

Comment:
I think that we don't actually require xcode for anything anymore. I don't currently have it installed and the DR tests pass anyway. IIRC homebrew automatically installs the command line tools. I think we should remove this file and see how that goes.

Reply:
Cool! I'll make a PR to remove it once this is merged.